engine/access/state_stream/engine.go

```diff
@@ -154,7 +147,7 @@ func (e *Engine) OnExecutionData(executionData *execution_data.BlockExecutionDat
 		Hex("block_id", logging.ID(executionData.BlockID)).
 		Msg("received execution data")
 
-	_ = e.execDataCache.Add(executionData.BlockID, executionData)
+	_ = e.execDataCache.Add(executionData)
```

Comment:
Logging the returned `Add()` value may indicate debugging insights at production. For example, add will always fail if the cache is misconfigured to use no-ejection.

Reply:
[2b9bcea](https://github.com/onflow/flow-go/pull/4197/commits/2b9bcea5691ef5ce712a64adfc0bb0995f8ed01b)